Src/FluentAssertions/Execution/AssertionFailedException.cs

```diff
@@ -16,6 +17,7 @@ public AssertionFailedException(string message)
     {
     }
 
+    [ExcludeFromCodeCoverage]
```

Comment:
How can I test this? Since the exception message cannot be serialized at all, because of some issues in the internals (as I stated [here](https://github.com/fluentassertions/fluentassertions/pull/2746#discussion_r1732494992)

Reply:
Is this a breaking change then?

Reply:
Technically yes, so we should mention it in the `releases.md`, but in reality nobody will be affected by this.